src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -9,7 +9,8 @@ public static class MetricEventNames
         public const string HostStartupLatency = "host.startup.latency";
 
         // function level events
-        public const string FunctionInvokeByTriggerFormat = "function.invoke.{0}";
```

Comment:
However, we could leave a function.invoke metric as well as an easy top level count of how many invocations are occurring, regardless of trigger type. Thoughts? We can count those per function app/sub.